<ui version="4.0" >
 <class>SuperCellDialog</class>
 <widget class="QDialog" name="SuperCellDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Super Cell Builder</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>195</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Super Cell Size</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="aRepeatLabel" >
         <property name="text" >
          <string>Miller h:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="millerHSpinBox" >
         <property name="minimum" >
          <number>-20</number>
         </property>
         <property name="maximum" >
          <number>20</number>
         </property>
         <property name="value" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="bRepeatLabel" >
         <property name="text" >
          <string>Miller k:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="millerKSpinBox" >
         <property name="minimum" >
          <number>-20</number>
         </property>
         <property name="maximum" >
          <number>20</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="cRepeaLabel" >
         <property name="text" >
          <string>Miller l:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QSpinBox" name="millerLSpinBox" >
         <property name="minimum" >
          <number>-20</number>
         </property>
         <property name="maximum" >
          <number>20</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SuperCellDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel" >
     <x>125</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SuperCellDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel" >
     <x>125</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
